docs/DeveloperGuide.md

```diff
@@ -404,23 +404,35 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes at step 1.
 
-**Use case: Filter persons by tag**
+**Use case: Find persons by name and/or tag**
 
 **MSS**
 
-1.  User requests to list persons with a specific tag description
+1.  User requests to list persons with a specific name and/or tag
```

Comment:
"and/or" somehow is confusing for me. I think your code implementes in the "or" way? For example someone enters `find n/alex t/friends`. As long as someone has `Alex` in name **or** `friends` in the tag, that person will be shown in the result. But I feel like if it is written as "by name or tag", it is also kind of misleading. So in short I think this needs some change in wordings

Reply:
Yeap I agree with this. I will find a suitable naming for it. Thanks for pointing it out!

Reply:
Something like this?
![image](https://user-images.githubusercontent.com/77256455/157607673-f6bb4b02-7825-471c-821b-9c2d56339bdb.png)

Reply:
LGTM